PersonApi/Startup.cs

```diff
@@ -132,6 +134,8 @@ public void ConfigureServices(IServiceCollection services)
                 var xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
                 if (File.Exists(xmlPath))
                     c.IncludeXmlComments(xmlPath);
+
+                services.Configure<AwsConfiguration>(options => Configuration.GetSection("AWS").Bind(options));
```

Comment:
Is this bit supposed to be within the `AddSwaggerGen` call?

Reply:
I don't think so. This simply provides a strongly typed class for appsettings that can then be injected.